### DISCUSSION TITLE
how to boldface matrix delimeters?

### DISCUSSION THREAD

**Discussion by stapeleliz** [KaTeX/KaTeX#3769] [Q&A]
When I want a bolded matrix (matrix, array, bmatrix, etc), I have to use \boldsymbol on each and every entry of the matrix; using \boldsymbol on the entire matrix doesn't affect the entries inside it.

However, whether or not I've got another \boldmatrix wrapped around the entire matrix, I can't seem to get the delimiters to be bolded. What am I missing?

Thank you!

Eliz.

**Comment by edemaine** [comment]
I just checked LaTeX, and `\boldsymbol` wrapped around the matrix *should* affect all the entries. This is a KaTeX bug (feel free to open an issue).

LaTeX doesn't seem to use different `\left`/`right` brackets when in `\boldsymbol` mode; probably a limitation of `\boldsymbol`. I agree that this would be useful, but it would require specific bold bracket symbols which we currently don't have.

**Reply by stapeleliz** [reply]
Thank you for checking into this for me!

Eliz.
